example_site/src/components/AccountPage.js

```diff
@@ -18,21 +26,82 @@ export default function AccountPage() {
     }
   );
 
+  const [isUpdatingAccount, setIsUpdatingAccount] = useState(false);
+
+  const onSubmitUpdateAccountForm = async (formData) => {
+    if (!(formData.emailAddress || formData.externalID)) {
+      openSnackbar("Please provide an email address and external ID.");
+      return;
+    }
+
+    try {
+      await waymarkInstance.updateAccountInfo(formData);
+
+      console.log("Account sucessfully updated");
+      openSnackbar("Account sucessfully updated");
+
+      const account = await waymarkInstance.getAccountInfo();
```

Comment:
We can make the `updateAccountInfo` call resolve with the account which would remove the need for another call. I did that for `loginAccount` as well.